test/build_version_test.dart

```diff
@@ -31,8 +31,8 @@ void main() {
         buildVersion(), _createPackageStub({'name': 'pkg', 'version': '1.0.0'}),
         outputs: {
           'pkg|lib/src/version.dart': (r'''
-// Generated code. Do not modify.
-const packageVersion = '1.0.0';
+/// Generated code. Do not modify.
```

Comment:
To avoid linter warnings : `public_member_api_docs` and `type_annotate_public_apis`.